src/igvfd/schemas/matrix_file.json

```diff
@@ -132,22 +134,31 @@
                 "igvf_utils": "cell"
             }
         },
-        "dimension2": {
-            "title": "Dimension 2",
-            "description": "Second dimension of the matrix.",
-            "type": "string",
-            "enum": [
-                "cell",
-                "fragment",
-                "gene",
-                "time",
-                "treatment",
-                "variant",
-                "genomic position"
-            ],
+        "dimension_y": {
+            "title": "Dimension Y",
+            "description": "The secondary, tertiary....n levels of dimensions of the matrix.",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Type",
```

Comment:
I think "Type" is somewhat vague, would it make sense to call this "Dimension"?

Reply:
Oh sorry, totally a typo :)